charts/kargo/values.yaml

```diff
@@ -112,11 +112,17 @@ api:
     ## @param api.oidc.cliClientID The client ID for the OIDC client used by CLI (optional). Needed by some OIDC providers (such as Dex) that require a separate Client ID for web app login vs. CLI login (`http://localhost`). If Dex is enabled, this value will be ignored and cli client ID will be automatically configured. If Dex is not enabled, and a different client app is configured for localhost CLI login, this should be the client ID configured in the IdP.
     cliClientID:
 
+    globalServiceAccounts:
+      ## @param api.oidc.globalServiceAccounts.enabled Whether to enable the api to look for service accounts in specific namespaces used to share service accounts.
+      enabled: false
```

Comment:
See [this related comment](https://github.com/akuity/kargo/pull/1190/files#r1411408832)